src/lib/RichText/Converter/EzNoNamespace.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\FieldTypeRichText\RichText\Converter;
+
+use DOMDocument;
+use Ibexa\Bundle\FieldTypeRichText\Command\MigrateNamespacesCommand;
+use Ibexa\Contracts\FieldTypeRichText\RichText\Converter;
+
+class EzNoNamespace implements Converter
+{
+    public function convert(DOMDocument $xmlDoc)
+    {
+        $xml = $xmlDoc->saveXML();
+        $xml = MigrateNamespacesCommand::migrateNamespaces($xml);
+        $xmlDoc->loadXML($xml);
+
+        return $xmlDoc;
```

Comment:
I agree that it will degrade performance for all customers, but the problem is also explained in other comments in this PR:
Upgrading from 3.x to 4.x will for customers with lots of content take hours (  that means hours of downtime, or at least hours where content moderation/editing is not possible).

Migrating the namespaces will add hours of additional downtime for such clients. Thus, it was requested to have the possibility to do this in two steps ( so two shorter periods of downtime instead of one very long period ).

If we refuse to support on-the-fly migration for a short period ( I am thinking one or two sub-releases ), upgrade process will be significant more painful for some customers.

That said, it might be an alternative to have on-the-fly conversions enabled/disabled in config, or making support for it in a custom bundle that can be installed optionally. 
The only change needed in richtext bundle for making the later pretty easy is to keep the Aggregate converter for ichtext->xhtml5edit which this PR already introduces

Reply:
@vidarl We decided to takie this PR over and finish it. @ciastektk will work on that.

Reply:
We're not going to accept the solution which degrades performance, while it could be avoided by deterministic one time change. The solution simply needs to work quickly through a simple query.

> Upgrading from 3.x to 4.x will for customers with lots of content take hours ( that means hours of downtime, or at least hours where content moderation/editing is not possible).

Live upgrade script needs to be quick. It also should be tested on a fresh snapshot of production database beforehand.

> If we refuse to support on-the-fly migration for a short period ( I am thinking one or two sub-releases ), 

This is formally not possible due to BC. You cannot change the behavior of such complex Product back and forth within one major release.

Anyway, as @webhdx said we're going to take over this one, so don't worry about it :+1: